t/27-consoles-sshIucvconn.t

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/perl
+
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+use Test::Most;
+use Mojo::Base -strict, -signatures;
+use FindBin '$Bin';
+use lib "$Bin/../external/os-autoinst-common/lib";
+use OpenQA::Test::TimeLimit '5';
+use consoles::sshIucvconn;
+use Test::MockObject;
+use Test::MockModule;
+use testapi;
```

Comment:
Add Test::Warnings , see https://github.com/os-autoinst/os-autoinst/actions/runs/11738913278/job/32702352462?pr=2565#step:4:318

and better to use external imports first, then internal. And probably you don't need testapi. Or actually you shouldn't use it in a unit test for a console as testapi is more high level than a console

```suggestion
use Test::Most;
use Mojo::Base -strict, -signatures;
use Test::MockObject;
use Test::MockModule;
use FindBin '$Bin';
use lib "$Bin/../external/os-autoinst-common/lib";
use OpenQA::Test::TimeLimit '5';

use consoles::sshIucvconn;  # SUT
```

Reply:
Done. However testapi is needed.

Reply:
ok, I suggest to please use the comment `# SUT` as is done in other places, e.g. t/29-backend-ipmi.t